packages/core/src/indexing/__tests__/database-index-api.test.ts

```diff
@@ -1033,24 +1119,37 @@ describe('sqlite', () => {
 
   describe('indexModels', () => {
     test('populates the INDEXED_MODEL_CONFIG_TABLE_NAME table on indexModels()', async () => {
-      const modelsToIndex = [StreamID.fromString(STREAM_ID_A), Model.MODEL]
+      const modelToIndex = StreamID.fromString(STREAM_ID_A)
+      const modelToIndexArgs = {
+        model: modelToIndex,
+        indices: [
+          {
+            name: 'test_model_index',
+            fields: [
+              {
+                path: ['name'],
+              },
+              {
+                path: ['address'],
+              },
+            ],
+          },
+        ],
+      }
       const indexApi = new SqliteIndexApi(dbConnection, true, logger, Networks.INMEMORY)
       indexApi.setSyncQueryApi(new CompleteQueryApi())
       await indexApi.init()
-      await indexApi.indexModels(modelsToIndexArgs(modelsToIndex))
+      await indexApi.indexModels([modelToIndexArgs])
```

Comment:
Could you add tests to check the behavior when this API is called multiple times with the same arguments please?